views/index.etlua

```diff
@@ -1,19 +1,239 @@
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="section__content">
+            <p>
+                Lua in the browser means you can use coroutines to write beautiful asynchronous code:
+            </p>
+
+            <div class="center">
+                <div class="snippet">
+                    <code class="snippet__code lua">
+                        <pre>
+coroutine.wrap(function()
+    local data = fetch("/static/data/test.html")
+
+    print("Received: ", data)
+end)()
+                        </pre>
+                    </code>
+                    <div class="snippet__output"></div>
+                    <div class="snippet__actions">
+                        Try it: 
+                        <div class="snippet__actions__action" data-action="run">
+                            ▶
+                            <div class="snippet__source" data-lang="lua">
+                                local fetch = function(url)
+                                    local co = coroutine.running()
+
+                                    local promise = js.global:fetch(url)
+
+                                    promise["then"](promise, function(_, res)
+                                        local textPromise = res:text()
+                                        textPromise["then"](textPromise, function(_, text)
+                                            coroutine.resume(co, text)
+                                        end)
+                                    end)
+
+                                    local res = coroutine.yield()
+
+                                    return res
+                                end
+
+                                local snippetPrint = _G.print
+
+                                local co = coroutine.wrap(function()
+                                    local data = fetch("/static/data/test.html")
+
+                                    snippetPrint("Received: ", data)
+                                end)
+
+                                co()
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="section">
+        <div class="section__content">
+            <h2>Test it out !</h2>
+            <div class="repl">
+                <code id="fengari-console"></code>
+                <div class="fengari-input-container">
+                    <label id="fengari-prompt" for="fengari-input">→ </label>
+                    <textarea class="lua" id="fengari-input" rows="1" placeholder="Type some Lua here..."></textarea>
+                </div>
+            </div>
+
+            This REPL is itself <a href="https://github.com/fengari-lua/fengari.io/blob/master/static/lua/web-cli.lua">written in Lua</a>.
```

Comment:
I don't like links to `master` branch (might die in future). Could we have a build process that writes in the commit id?